twitterscraper/TwitterScraper.py

```diff
@@ -2,147 +2,166 @@
 import json
 import csv
 import logging
-import urllib2, urllib
-import random, time
+import urllib
```

Comment:
I think [this](http://stackoverflow.com/a/17510727) can make it compatible for python 2 and 3 simultaneously:

try:
    # For Python 3.0 and later
    from urllib.request import urlopen, Request
except ImportError:
    # Fall back to Python 2's urllib2
    from urllib2 import urlopen, Request

It seems to work for Python 2, but can you test if it also works for 3?